packages/form-js-editor/test/spec/FormEditor.spec.js

```diff
@@ -356,13 +393,15 @@ describe('createFormEditor', function() {
 });
 
 async function waitForFormEditorCreated(options) {
-  const form = createFormEditor(options);
+  const formEditor = createFormEditor(options);
+
+  window.formEditor = formEditor;

```

Comment:
Is this a left over debug utility or should it stay?

Reply:
Removed (cf. https://github.com/bpmn-io/form-js/pull/75/commits/fe8b7ac211663b10697c2f45f28c6647acdfae45).